samples/msgext-ai-sentiment-analysis/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs",
+    "source": "officeDev",
+    "title": "Sentiment analysis with messaging extension and Azure Open AI",
+    "shortDescription": "Sentiment analysis Using Azure Open AI through messaging extension.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-ai-sentiment-analysis/nodejs",
+    "longDescription": [
+      "This sample shows a feature where user can get sentiment analysis for posted messages in Teams chat using messaging extension and Azure Open AI."
+    ],
+    "creationDateTime": "2023-08-07",
+    "updateDateTime": "2023-08-07",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "javascript"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "express"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot, msgext, Azure Open AI, toolkit"
```

Comment:
Do we need bot??
what is msgext ???

Reply:
used bot to send welcome message when we add application in teams

Reply:
replaced msgext with **ME**